bin/cli.js

```diff
@@ -65,18 +63,13 @@ const cliFlags = yargs
     return true;
   }).argv;
 
-//Get all explicitly terminal set options, does not include url because url has no default option
```

Comment:
I think reason doing that is not to rewrite flags in config by default values. Idea about replacing flags in config by passed cli flags requires to do that. 

E.g. if you have flags in config.
```js
{
  {
    json: true,
    expectations: true
  }
}
```

If user didn't pass any cli flags
this `let options = Object.assign({}, config, {flags: cliFlags});` will rewrite config by default values. 

```js
{
  {
    json: false,
    expectations: false
  }
}
```